examples/seismic/acoustic/acoustic_example.py

```diff
@@ -43,7 +43,7 @@ def acoustic_setup(shape=(50, 50, 50), spacing=(15.0, 15.0, 15.0),
     src.coordinates.data[0, -1] = model.origin[-1] + 2 * spacing[-1]
 
     # Define receiver geometry (spread across x, just below surface)
-    rec = Receiver(name='rec', grid=model.grid, ntime=nt, npoint=nrec)
+    rec = Receiver(name='rec', grid=model.grid, time=time, npoint=nrec)
```

Comment:
``grid`` has a reference to time, namely ``grid.time_dim`` or something along these lines. Can you rather use that?

Reply:
same story below

Reply:
Isn't the problem that grid doesn't always have a time_dim?

Reply:
ok if you need a real value that's fine, but then calling it ``time`` is confusing because this term is also used everywhere to refer to the time ``:class:`Dimension```

Reply:
Renamed to time_values.